tempodb/encoding/vparquet/compactor.go

```diff
@@ -315,18 +316,27 @@ func estimateMarshalledSizeFromParquetRow(row parquet.Row) (size int) {
 }
 
 // countSpans counts the number of spans in the given trace in deconstructed
-// parquet row format. It simply counts the number of values for span ID, which
-// is always present.
-func countSpans(schema *parquet.Schema, row parquet.Row) (spans int) {
+// parquet row format and returns traceId.
+// It simply counts the number of values for span ID, which is always present.
+func countSpans(schema *parquet.Schema, row parquet.Row) (traceID string, spans int) {
+	traceIDColumn, found := schema.Lookup(TraceIDColumnName)
+	if !found {
+		return "", 0
+	}
+
 	spanID, found := schema.Lookup("rs", "ils", "Spans", "ID")
 	if !found {
-		return 0
+		return "", 0
 	}
 
 	for _, v := range row {
 		if v.Column() == spanID.ColumnIndex {
 			spans++
 		}
+
+		if v.Column() == traceIDColumn.ColumnIndex {
```

Comment:
Does this seems to be ok?
I've want to find `traceId` in a parquet row.

Reply:
i think this works :). add a test to confirm?

Reply:
Okay I'll add a test.

Reply:
done :)